packages/types/CHANGELOG.json

```diff
@@ -5,6 +5,10 @@
             {
                 "note": "Added ERC1155 revert reasons",
                 "pr": 1657
+            },
+            {
+                "note": "Add `RevertReason.SignatureInvalid` thrown by Coordinator",
+                "pr": 1705
```

Comment:
Not your fault but can you merge this CHANGELOG entry with the one below? We really need to write a CI test that fails if there are multiple entries in a CHANGELOG without timestamps. Gonna add it to the next Tech Debt Demolishion Sprint.